pkg/asset/release/release.go

```diff
@@ -0,0 +1,41 @@
+// Package release contains assets for the release image (also known
+// as the update payload).
+package release
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/installer/pkg/asset"
+)
+
+var (
+	defaultReleaseImage = "registry.svc.ci.openshift.org/openshift/origin-release:v4.0"
+)
+
+// Release is the pull-spec for the release image.
+type Release string
```

Comment:
nit: ReleaseImage

Reply:
> nit: ReleaseImage / Image

Changed to `ReleaseImage` with e189de1 -> 3a99f85.

Reply:
> Changed to `ReleaseImage` with e189de1 -> 3a99f85.

[golint][1]:

```
2019/03/08 09:10:21 Executing test golint
/go/src/github.com/openshift/installer/pkg/asset/release/release.go:18:6: type name will be used as release.ReleaseImage by other packages, and that stutters; consider calling this Image
/go/src/github.com/openshift/installer/pkg/asset/release/release.go:33:1: receiver name r should be consistent with previous receiver name i for ReleaseImage
Found 2 lint suggestions; failing.
```

So I've pushed 3a99f85b8 -> 327976e5e pivoting to `Image`.

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/1286/pull-ci-openshift-installer-master-golint/3155/build-log.txt